README.md

```diff
@@ -18,8 +18,9 @@ This project contains
 Software artifacts can be built using [Nix](https://nixos.org/nix/).
 
 * To build the Haskell project, run `nix-build -A haskell`.
-* To use the Haskell project, try `nix-shell -p "(import ./default.nix {}).haskellPackages.ghcWithPackages (pkgs: [pkgs.Simplicity])"`.
```

Comment:
This line above enters a shell with Simplicity already built (useful for manual testing the code in ghci), and thus shouldn't be eliminated as it is unrelated to your shell which enters an environment to build Simplicity.